vhdbuilder/packer/components.json

```diff
@@ -64,8 +64,8 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.35",
```

Comment:
we still have .35 as the version for Windows ... do we want to remove it here yet?

Reply:
I think this one is for linux, windows gets updated in vhdbuilder/packer/generate-windows-vhd-configuration.ps1